static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -17,7 +18,9 @@ import ParentLabel from 'sentry/views/settings/account/notifications/parentLabel
  * Which fine-tuning parts are grouped by project
  */
 export const isGroupedByProject = (notificationType: string): boolean =>
-  ['alerts', 'email', 'workflow', 'activeRelease'].includes(notificationType);
+  ['alerts', 'email', 'workflow', 'activeRelease', 'spikeProtection'].includes(
+    notificationType
+  );
```

Comment:
This should also be a constant so it's more easily found. Honestly I think the notification setting response from the API should be determining whether to group it by Project but that's a bit out of scope for now.